### ISSUE TITLE
React 0.14.0 compatibility

### ISSUE THREAD

**Issue by gregory90** [balloob/react-sidebar#18]
I'm seeing following errors:
```
Warning: require('react/addons') is deprecated. Access using require('react-addons-{addon}') instead.
```
Also when installing with `npm install react-sidebar`:
```
@my-project@1.0.0-rc1 /project
├── UNMET PEER DEPENDENCY react@0.14.0-rc1
└── react-sidebar@1.0.4
```

**Comment by balloob** [comment]
I have a [0.14 version branch](https://github.com/balloob/react-sidebar/tree/react-14). Won't merge till 0.14 has been officially released.

**Comment by Nitive** [comment]
Release version for React 14 please.
It had been released.

**Comment by balloob** [comment]
:+1:

**Comment by balloob** [comment]
React Sidebar 2.0 has been released which is compatible with React 0.14
